instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2ClientTest.groovy

```diff
@@ -496,10 +496,9 @@ abstract class AbstractAws2ClientTest extends InstrumentationSpecification {
   // the instrumentation to add Events for retries instead.
   def "timeout and retry errors not captured"() {
     setup:
-    def response = HttpResponse.delayed(HttpResponse.of(HttpStatus.OK), Duration.ofMillis(500))
```

Comment:
Oops! This can be changed to AggregatedHttpResponse instead if reintancing if preferred

Reply:
I wasn't able to figure out how to use AggregatedHttpResponse here.